exercises/secret-handshake/secret_handshake_test.go

```diff
@@ -28,7 +28,10 @@ func TestHandshake(t *testing.T) {
 }
 
 func BenchmarkHandshake(b *testing.B) {
+	var j uint
 	for i := 0; i < b.N; i++ {
-		Handshake(31)
+		for j = 0; j < 32; j++ {
```

Comment:
Consider, more simply:
> for j := uint(0); j < 32; j++ {

since j is needed only inside the for loop, and then we can drop the var outside.